src/ExceptionReporting/ExceptionReporter.NET-sdk.csproj

```diff
@@ -33,6 +33,25 @@
     <Reference Include="System.Management" />
     <Reference Include="System.Windows.Forms" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
So I suppose we don't want to repeat doing this in `ExceptionReporter.NET-traditional.sln` so we better delete that old solution - I think VS2019 or greater is required to read the new project style (?) ... something like that.

Reply:
>  I think VS2019 or greater is required to read the new project style (?)

VS2019 supports both project style.

> So I suppose we don't want to repeat doing this in ExceptionReporter.NET-traditional.sln so we better delete that old solution

I vote to delete ExceptionReporter.NET-traditional.sln, but you decide

Reply:
Yeah thanks, I might do that in another commit after the merge and let this go through